modules/serverless-kafka-sasl.adoc

```diff
@@ -27,15 +27,21 @@ Red Hat recommends to enable TLS in addition to SASL.
 [source,terminal]
 ----
 $ oc create secret --namespace <namespace> generic <kafka_auth_secret> \
+  --from-literal=protocol="SASL_SASL" \
   --from-file=ca.crt=caroot.pem \
   --from-literal=password="SecretPassword" \
-  --from-literal=saslType="SCRAM-SHA-512" \
+  --from-literal=sasl.mechanism="SCRAM-SHA-512" \
   --from-literal=user="my-sasl-user"
 ----
 +
 [IMPORTANT]
 ====
-Use the key names `ca.crt`, `password`, and `saslType`. Do not change them.
+Use the key names `protocol`, `ca.crt`, `password`, and `sasl.mechanism`. Do not change them.
+====
+
+[NOTE]
+====
+If TLS is not required, do not specify the `ca.crt` key and change the `protocol` to `SASL_PLAINTEXT`.
```

Comment:
This explains the sentence here :https://github.com/openshift/openshift-docs/pull/46416/files#diff-2d568646d07f4762ad836f963b26119cfb38d2d4220bd42f75d6e457118dc18dR15

Reply:
There shouldn't be two notes next to each other like this, it should probably be added as a callout if it's explaining something in the code block above?